src/include/common/cpu/rpp_cpu_simd_load_store.hpp

```diff
@@ -1703,6 +1703,29 @@ inline void rpp_load24_f32pkd3_to_f32pln3_mirror_avx(Rpp32f *srcPtr, __m256 *p)
     p[2] = _mm256_permutevar8x32_ps(p[2], pxMask); /* shuffle as B08-B01 */
 }
 
+inline void rpp_load24_f16pkd3_to_f32pln3_mirror_avx(Rpp16f *srcPtr, __m256 *p)
+{
+    __m128 p128[8];
+    __m256i pxMask = _mm256_setr_epi32(7, 6, 5, 4, 3, 2, 1, 0);
+    p128[0] = _mm_cvtph_ps(_mm_castps_si128(_mm_loadu_ps(reinterpret_cast<Rpp32f *>(srcPtr)))); /* loads R01|G01|B01|R02 */
```

Comment:
Is it possible to extend this linline comment to add a quick reason why SSE registers are used here instead of AVX2 - specify reasons like pkd3->pln3 cross-lane shuffling is inefficient in AVX2

Reply:
We should ideally add it over whole segmetns of code like this that fall under our AVX2 processing but are internally also using SSE for very specific reasons